layers/+spacemacs/spacemacs-evil/funcs.el

```diff
@@ -84,16 +84,20 @@ Otherwise, revert to the default behavior (i.e. enable `evil-insert-state')."
        (or (not (fboundp 'evil-mc-get-cursor-count))
            (eq (evil-mc-get-cursor-count) 1))))
 
-(defun spacemacs/evil-mc-paste-after (&optional count register)
-  "Disable paste transient state if there is more that 1 cursor."
-  (interactive "p")
-  (if (spacemacs//paste-transient-state-p)
-      (spacemacs/paste-transient-state/evil-paste-after)
-    (evil-paste-after count (or register evil-this-register))))
+(defun spacemacs/evil-mc-paste-after (fn &rest args)
+   "Disable paste transient state if there is more that 1 cursor."
```

Comment:
The extra space before these lines should probably be removed. Two spaces seem to be default. At least when one auto indents the paragraph with `evil-indent`: `= i p`. The `spacemacs/evil-mc-paste-before` function only has two spaces, so they should probably match. Otherwise these lines will show up unexpectedly (for a git beginner), if the whole file is auto indented.

Additionally there's a small typo in the doc-string. 
"that" should be changed to "than" resulting in:
"Disable paste transient state if there is more than 1 cursor."
the same typo occurs in both functions.

Or should the typo be fixed in a separate PR?